rust/relay/src/main.rs

```diff
@@ -83,8 +83,6 @@ async fn main() -> Result<()> {
 
     let server = Server::new(args.public_ip4_addr, make_rng(args.rng_seed));
 
-    tracing::info!("Relay auth secret: {}", server.auth_secret());
```

Comment:
Why remove that?

Reply:
I think since it's a secret, even if the risk from leakage is low we should treat it as one and AFAIK there's not much use for printing this on console?